package.nls.json

```diff
@@ -418,6 +427,41 @@
     "AWS.apig.remoteInvoke.noApiResource": "Select an API Resource",
     "AWS.apig.remoteInvoke.noMethod": "Select a HTTP method",
     "AWS.apig.selectStage": "Select an API stage",
+    "AWS.iot.createThing.prompt": "Enter a new Thing name",
+    "AWS.iot.createThing.placeHolder": "Thing Name",
+    "AWS.iot.createThing.success": "Created Thing {0}",
+    "AWS.iot.createThing.error": "Failed to create Thing: {0}",
+    "AWS.iot.deleteThing.prompt": "Are you sure you want to delete Thing {0}?",
+    "AWS.iot.deleteThing.success": "Deleted Thing {0}",
+    "AWS.iot.deleteThing.error": "Failed to delete Thing {0}",
+    "AWS.iot.deleteCert.prompt": "Are you sure you want to delete Certificate {0}?",
+    "AWS.iot.deleteCert.success": "Deleted Certificate {0}",
+    "AWS.iot.deleteCert.error": "Failed to delete Certificate {0}",
+    "AWS.iot.deleteCert.activeError": "Active certificates cannot be deleted",
+    "AWS.iot.deleteCert.retrieveError": "Failed to retrieve {0} attached to certificate",
+    "AWS.iot.deleteCert.attachedError": "Certificate has attached {0}",
+    "AWS.iot.deletePolicy.prompt": "Are you sure you want to delete Policy {0}?",
+    "AWS.iot.deletePolicy.success": "Deleted Policy {0}",
+    "AWS.iot.deletePolicy.error": "Failed to delete Policy {0}",
+    "AWS.iot.activateCert.prompt": "Are you sure you want to activate certificate {0}?",
+    "AWS.iot.activateCert.confirm": "Activate",
+    "AWS.iot.activateCert.success": "Activated {0}",
+    "AWS.iot.activateCert.error": "Failed to activate {0}",
+    "AWS.iot.deactivateCert.prompt": "Are you sure you want to deactivate certificate {0}?",
+    "AWS.iot.deactivateCert.confirm": "Deactivate",
+    "AWS.iot.deactivateCert.success": "Deactivated {0}",
+    "AWS.iot.deactivateCert.error": "Failed to deactivate {0}",
+    "AWS.iot.revokeCert.prompt": "Are you sure you want to revoke certificate {0}?",
+    "AWS.iot.revokeCert.confirm": "Revoke",
+    "AWS.iot.revokeCert.success": "Revoked {0}",
+    "AWS.iot.revokeCert.error": "Failed to revoke {0}",
+    "AWS.iot.detachCert.prompt": "Are you sure you want to detach certificate from Thing {0}?",
+    "AWS.iot.detachCert.confirm": "Detach",
+    "AWS.iot.detachCert.success": "Detached {0}",
+    "AWS.iot.detachCert.error": "Failed to detach {0}",
+    "AWS.iot.detachPolicy.prompt": "Are you sure you want to detach policy {0}?",
+    "AWS.iot.detachPolicy.success": "Detached {0}",
+    "AWS.iot.detachPolicy.error": "Failed to detach {0}",
```

Comment:
So, these entries are actually only necessary for things not calling into the NLS `localize` function, such as references in `package.json` for things like command names etc. I have gotten burned on this too.